mlflow/server/__init__.py

```diff
@@ -76,6 +77,10 @@ def serve_model_version_artifact():
 def serve_get_metric_history_bulk():
     return get_metric_history_bulk_handler()
 
+# Serve the "experiments/search-datasets" route.
+@app.route(_add_static_prefix("/ajax-api/2.0/mlflow/experiments/search-datasets"))
+def serve_search_datasets():
+    return search_datasets_handler()
```

Comment:
Can we add a test case to `test_rest_tracking.py` to verify that this API works?